### DISCUSSION TITLE
my drawings for live quest stories! (0-3)

### DISCUSSION THREAD

**Discussion by utkuasd** [stellar/sorobanathon#100] [General]
hey! here's my effort to draw the live quest series stories. i know they're not too nice but i tried my best... I'll post rest of the chapters when I've finished them. thanks!

prologue: https://litemint.com/items/GBODDSY4LX5DRQ6ZD2TLG3U5YZ7XTS6KE76RDVJJLW7WRAOMAU2JBKWB/NFT
chapter 1: https://litemint.com/items/GCQWOONFZ2AFKACUTRO2KJDAYJRFZXVC2XPIGISFRLO6B32JIKSIISGC/NFT
chapter 2: https://litemint.com/items/GAX2FEY35VIHHD6EB22HUMVSWITAGJNIOXKZ3XJVVAUBGYEZJPLK7C5Z/NFT
chapter 3 https://litemint.com/items/GAOZPCTENX5GF6QCTSIWWXKQRLIKSCBNIQUTNQHHYEXX2R7TBARCWLV6/NFT

**Comment by stellarjohn** [comment]
Hi utkuasd!

Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 1200 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John
